draft-ietf-webpush-protocol.xml

```diff
@@ -599,21 +599,32 @@ DATA         [stream 4] +END_STREAM
           at least once, the user agent MUST acknowledge receipt of the message
           by performing a HTTP DELETE on the push message resource.
         </t>
-
         <figure>
           <artwork type="inline">
             <![CDATA[
 DELETE /d/qDIYHNcfAIPP_5ITvURr-d6BGtYnTRnk HTTP/1.1
 Host: push.example.net
-
 ]]>
           </artwork>
         </figure>
-
         <t>
-          If the application has requested a delivery receipt, the push server
-          MUST deliver a response to the application server monitoring the
-          receipt subscription resource.
+          If the push server receives the acknowledgement and the application
+          has requested a delivery receipt, the push server MUST deliver a success
+          response to the application server monitoring the receipt subscription
+          resource.
+        </t>
+        <t>
+          If the push server does not receive the acknowledgement within a
+          reasonable amount of time, then the message is considered to be not yet
+          delivered. The push server SHOULD continue to retry delivery of the message
+          until its advertised expiration.
+        </t>
+        <t>
+          If the push server ceases to retry delivery of the message prior to its
+          advertised expiration (perhaps due to an unresponsive user agent) and the

```

Comment:
Style note: I would move the parenthetical to a lead-in sentence and establish that the push service (not push server) might have several reasons for giving up, including unresponsive UA and operational constraints.